diff --git a/node_modules/@capacitor-firebase/authentication/Package.swift b/node_modules/@capacitor-firebase/authentication/Package.swift
index 909c7f0..08dbce7 100644
--- a/node_modules/@capacitor-firebase/authentication/Package.swift
+++ b/node_modules/@capacitor-firebase/authentication/Package.swift
@@ -12,8 +12,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "8.0.0"),
         .package(url: "https://github.com/firebase/firebase-ios-sdk.git", .upToNextMajor(from: "12.7.0")),
-        .package(url: "https://github.com/google/GoogleSignIn-iOS", from: "9.0.0"),
-        .package(url: "https://github.com/facebook/facebook-ios-sdk.git", from: "18.0.0")
+        .package(url: "https://github.com/google/GoogleSignIn-iOS", from: "9.0.0")
     ],
     targets: [
         .target(
@@ -23,14 +22,11 @@ let package = Package(
                 .product(name: "Cordova", package: "capacitor-swift-pm"),
                 .product(name: "FirebaseAuth", package: "firebase-ios-sdk"),
                 .product(name: "FirebaseCore", package: "firebase-ios-sdk"),
-                .product(name: "GoogleSignIn", package: "GoogleSignIn-iOS"),
-                .product(name: "FacebookCore", package: "facebook-ios-sdk"),
-                .product(name: "FacebookLogin", package: "facebook-ios-sdk")
+                .product(name: "GoogleSignIn", package: "GoogleSignIn-iOS")
             ],
             path: "ios/Plugin",
             swiftSettings: [
-                .define("RGCFA_INCLUDE_GOOGLE"),
-                .define("RGCFA_INCLUDE_FACEBOOK")
+                .define("RGCFA_INCLUDE_GOOGLE")
             ]),
         .testTarget(
             name: "FirebaseAuthenticationPluginTests",
